src/Sulu/Bundle/EventLogBundle/Infrastructure/Sulu/Admin/EventLogAdmin.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\EventLogBundle\Infrastructure\Sulu\Admin;
+
+use Sulu\Bundle\AdminBundle\Admin\Admin;
+use Sulu\Bundle\AdminBundle\Admin\Navigation\NavigationItem;
+use Sulu\Bundle\AdminBundle\Admin\Navigation\NavigationItemCollection;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewBuilderFactoryInterface;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewCollection;
+use Sulu\Bundle\EventLogBundle\Domain\Model\EventRecordInterface;
+use Sulu\Component\Localization\Manager\LocalizationManagerInterface;
+use Sulu\Component\Security\Authorization\PermissionTypes;
+use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
+
+class EventLogAdmin extends Admin
+{
+    const SECURITY_CONTEXT = 'sulu.event_log.event_records';
+
+    const LIST_VIEW = 'sulu_event_log.event_records.list';
+
+    /**
+     * @var ViewBuilderFactoryInterface
+     */
+    private $viewBuilderFactory;
+
+    /**
+     * @var SecurityCheckerInterface
+     */
+    private $securityChecker;
+
+    /**
+     * @var LocalizationManagerInterface
+     */
+    private $localizationManager;
+
+    public function __construct(
+        ViewBuilderFactoryInterface $viewBuilderFactory,
+        SecurityCheckerInterface $securityChecker,
+        LocalizationManagerInterface $localizationManager
+    ) {
+        $this->viewBuilderFactory = $viewBuilderFactory;
+        $this->securityChecker = $securityChecker;
+        $this->localizationManager = $localizationManager;
+    }
+
+    public function configureNavigationItems(NavigationItemCollection $navigationItemCollection): void
+    {
+        if (!$this->securityChecker->hasPermission(static::SECURITY_CONTEXT, PermissionTypes::VIEW)) {
+            return;
+        }
+
+        $eventLogNavigationItem = new NavigationItem('sulu_event_log.event_log');
```

Comment:
not sure how we should call this in the frontend yet? @sulu/core-team 

on one hand, i think `activity` would be a nice term. but on the other hand we dont use that term in the backend anywhere and i really want to avoid a confusion similar to the differences between `account`/`organization` 🙈

Reply:
as we renamed the bundle to `SuluActivityBundle` in #6003, we can use `Activity` now 🙂